modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1570,10 +1570,11 @@
     <string name="end_of_year_launch_modal_title">Your Year in Podcasts</string>
     <string name="end_of_year_launch_modal_image_title">Year in Podcasts</string>
     <string name="end_of_year_launch_modal_summary">See your top podcasts, categories, listening stats and more. Share with friends and shout out your favorite creators!</string>
-    <string name="end_of_year_launch_modal_primary_button_title">View My 2022</string>
+    <string name="end_of_year_launch_modal_primary_button_title">View My 2023</string>
     <string name="end_of_year_launch_modal_secondary_button_title">Not Now</string>
     <string name="end_of_year_prompt_card_title" translatable="false">@string/end_of_year_launch_modal_title</string>
-    <string name="end_of_year_prompt_card_summary">See your top podcasts, categories, listening stats and more.</string>
+    <string name="end_of_year_prompt_card_summary">See your listening stats, top podcasts, and more.</string>
+    <string name="end_of_year_pocket_casts_playback">Pocket Casts Playback</string>
```

Comment:
David asked that we not translate this, so I think we want this to be marked as not translatable. 

Related: https://github.com/Automattic/pocket-casts-android/pull/1490

Reply:
Removed in https://github.com/Automattic/pocket-casts-android/pull/1503/commits/97ba624d8c70c0b187c31a94c1c556a7b3691b30